d20potz.py

```diff
@@ -312,105 +317,79 @@ async def sendCards(
     await context.bot.send_media_group(chat_id=chat_id, media=media_list)
 
 
-async def getAllPlayerCards(update: Update, context: ContextTypes.DEFAULT_TYPE):
-    currentPlayer = getPlayerById(
-        update.effective_chat.id, getCurrentPlayerId(update.effective_chat.id)
-    )
-    playerName = currentPlayer
-    if len(update.message.text.split()) > 2:
-        await context.bot.send_message(
-            chat_id=update.effective_chat.id, text="Usage: /getallcards <player>"
-        )
-    if len(update.message.text.split()) == 2:
-        playerName = update.message.text.split()[1]
+async def cardsCommand(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    params = update.message.text.split()
+    chat_id = update.effective_chat.id
 
-    playerCards = CARDS[playerName.lower()]
-    if len(playerCards) == 0:
+    if len(params) < 3:
         await context.bot.send_message(
-            chat_id=update.effective_chat.id,
-            text="{} has no cards.".format(getSpelling(playerName)),
+            chat_id=chat_id, text="Usage: /cards <player> <subcommand>"
         )
-    else:
-        await sendCards(playerName, playerCards, update.effective_chat.id, context)
-
-
-def get_cards_in_order(player: str):
-    return sorted(CARDS[player])
-
-
-async def processCards(update: Update, context: ContextTypes.DEFAULT_TYPE):
-    async def send_to_chat(message: str):
-        await context.bot.send_message(chat_id=update.effective_chat.id, text=message)
+        return
 
-    command = update.message.text.lower().removeprefix("/cards").lstrip()
-    if not command:
-        await send_to_chat(
-            f"Usage: /cards (choose|retire|list|names) <card|card_index> [player]"
+    player_name = params[1].lower()
```

Comment:
You already checked that you have at least 3 params, can now just do:
```suggestion
    player_name, sub_command = params[:2]
```